sql/step_XX_drop_adm_deleted.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE patient DROP COLUMN pat_deleted;
```

Comment:
The file name looks wrong `step_XX....`    and the contents of this file is exactly the same as the next sql file